packages/mds-audit/api.ts

```diff
@@ -664,12 +685,11 @@ function api(app: express.Express): express.Express {
 
       try {
         const response = await getVehicles(skip, take, url, req.query, bbox, strict, provider_id)
-        return res.status(200).send(response)
+        return res.status(200).send({ version: res.locals.version, ...response })
```

Comment:
Do we want the result to be spread at the same nesting level as `version`? What about something like this
`res.status(200).send({ version: res.locals.version, response })`?

Edit: I see that this is just riffing off of what was present previously. Perhaps this could be revisited in a subsequent PR.

Reply:
The response body should follow the precedent set by the provider API spec. This could result in a breaking change though.